### PR TITLE
Update the Chinese translation for the beta version

### PR DESCRIPTION
**Pull request by CrewCyan**
Update this translation in Chinese so that we can better translate it in the future

### CODE REVIEW

**Comment by LezaiYa** [comment]
We have also added some translations for traditional Chinese

File: Roles/Impostor/Swooper.cs [hunk 1]
```diff
@@ -175,7 +174,6 @@ public static string GetHudText(PlayerControl pc)
 
     public static bool OnCheckMurder(PlayerControl killer, PlayerControl target)
     {
-        if (Medic.ProtectList.Contains(target.PlayerId)) return true;
```
Comment (Tommy-XL) [inline Roles/Impostor/Swooper.cs:178]: Translation removes some code

Comment (LezaiYa) [reply Roles/Impostor/Swooper.cs:178]: Okay, I'll fix it

**Comment by Tommy-XL** [comment]
Also, need rename ClientOptionItem.cs.cs to ClientOptionItem.cs

File: Roles/Impostor/Swooper.cs (hunk 1 above)
Comment (CrewCyan) [reply Roles/Impostor/Swooper.cs]: LezaiYa and I will continue to be responsible for translating Simplified Chinese and Traditional Chinese

**Comment by LezaiYa** [comment]
> 另外，需要将ClientOptionItem.cs.cs重命名为ClientOptionItem.cs

I have already renamed this file

**Comment by vdpcool** [comment]
I have renamed this file when TOHER is v2.4.2 :neutral_face:

**Comment by NetherDragonTw** [comment]
> We have also added some translations for traditional Chinese

我可以插個話嗎 （
我是前TOHE繁中翻譯員 鴿了太久了 看看這個暑假能不能搞個翻譯出來
(btw我之前跟Loonie pr過但是他不merge)

**Comment by LezaiYa** [comment]
> > We have also added some translations for traditional Chinese
> 
> 我可以插個話嗎 （ 我是前TOHE繁中翻譯員 鴿了太久了 看看這個暑假能不能搞個翻譯出來 (btw我之前跟Loonie pr過但是他不merge) 不過感謝你們給我一些翻譯靈感 也謝謝幫繁中的翻譯生態圈更進一步：D

那繁體中文的翻譯就交給你了

**Comment by vdpcool** [comment]
Thank you guys who are trans Chinese ;-;
Im very very miss Karp.

**Comment by CrewCyan** [comment]
> > We have also added some translations for traditional Chinese
> 
> 我可以插個話嗎 （ 我是前TOHE繁中翻譯員 鴿了太久了 看看這個暑假能不能搞個翻譯出來 (btw我之前跟Loonie pr過但是他不merge) 不過感謝你們給我一些翻譯靈感 也謝謝幫繁中的翻譯生態圈更進一步：D (如果能搞個crowdin出來那就更好了)

好的，接下來我和LezaiYa將全面負責簡體中文的翻譯，翻譯是一個很辛苦的過程，希望我們共同努力，讓翻譯更優質

**Pull request closed without merging**